cmake/cmakepp_lang/map/map.cmake

```diff
@@ -1,3 +1,8 @@
+#[[[ @module
+# Provides an easy way for the user to include the various functions used 
+# to create and maintain an instance of a CMakePPLang map.
+#]]
```

Comment:
Can you put the doc comments for the module under the license? Technically I think they aren't licensed if you put them above it.

Reply:
Oh yes, I misread the documentation and thought they had to be at the very top of the file. Comments can be above them.